app/electron/main.js

```diff
@@ -126,37 +126,44 @@ let serverProcess = null;
 let intentionalQuit = false;
 let serverProcessQuit = false;
 
+/**
+ * add some error handlers to the serverProcess.
+ * @param  {ChildProcess} serverProcess to attach the error handlers to.
+ */
+const attachServerEventHandlers = (serverProcess) => {
```

Comment:
We've been using `function ...` instead of the arrow functions.
Maybe we should move to arrows functions but for consistency for now, let's keep the functions.

Reply:
Ok, thanks. It's done.